osgp/platform/osgp-adapter-domain-core/src/test/java/org/opensmartgridplatform/adapter/domain/core/application/services/AdHocManagementServiceTest.java

```diff
@@ -0,0 +1,76 @@
+package org.opensmartgridplatform.adapter.domain.core.application.services;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+import org.opensmartgridplatform.adapter.domain.core.infra.jms.core.OsgpCoreRequestMessageSender;
+import org.opensmartgridplatform.domain.core.entities.Device;
+import org.opensmartgridplatform.domain.core.entities.Organisation;
+import org.opensmartgridplatform.domain.core.exceptions.UnknownEntityException;
+import org.opensmartgridplatform.domain.core.services.DeviceDomainService;
+import org.opensmartgridplatform.domain.core.services.OrganisationDomainService;
+import org.opensmartgridplatform.shared.exceptionhandling.ComponentType;
+import org.opensmartgridplatform.shared.exceptionhandling.FunctionalException;
+import org.opensmartgridplatform.shared.infra.jms.RequestMessage;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class AdHocManagementServiceTest{
+
+    @Mock
+    private DeviceDomainService deviceDomainService;
+    
+    @Mock
+    private OsgpCoreRequestMessageSender osgpCoreRequestMessageSender;
+
+    @Mock
+    private Device device;
+
+    @Mock
+    private OrganisationDomainService organisationDomainService;
+
+    
```

Comment:
I see all empty lines have been removed. I think that is fine, but to avoid misunderstanding a single empty line is not an issue if you want to create some visible separation, just avoid multiple empty lines.